amgut/lib/data_access/ag_data_access.py

```diff
@@ -267,6 +267,13 @@ def getAGHandoutKitDetails(self, supplied_kit_id):
 
         return kit_details
 
+    def getAGHandoutKitIDsAndPasswords(self):
+        sql = "SELECT kit_id, password FROM ag_handout_kits"
+        cur = self.get_cursor()
+        cur.execute(sql)
+
+        return [(i[0], i[1]) for i in cur.fetchall()]
```

Comment:
Is explicit tuple conversion necessary? It already comes out as a list of lists, so you can `return curr.fetchall()` and get the same layout, just inner list instead of tuple.